### PR TITLE
added cheatsheet to README

### PR DESCRIPTION
**Pull request by hritikb**

### CODE REVIEW

**Comment by hritikb** [comment]
I have skipped some advanced commands. Let me know if I need to add or change something.

File: README.md [hunk 1]
```diff
@@ -13,3 +13,37 @@
 ## Apendix
```
Comment (JaydevSR) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -13,3 +13,37 @@
...
 
 ### Cheatsheet of git commands
```
Comment (JaydevSR) [inline README.md]:
```suggestion
## Cheatsheet of git commands
```

File: README.md [hunk 3]
```diff
@@ -13,3 +13,37 @@
...
+
+#### For setup and initialization
```
Comment (JaydevSR) [inline README.md]:
```suggestion
### For setup and initialization
```

File: README.md [hunk 4]
```diff
@@ -13,3 +13,37 @@
...
+**git init**: initialize an existing directory as a Git repository<br>
+**git clone \[url\]**: retrieve an entire repository from a hosted location via URL<br>
+**git status**: show modified files in working directory, staged for your next commit<br>
+**git add \[file\]**: add a file as it looks now to your next commit (stage)<br>
+**git reset \[file\]**: unstage a file while retaining the changes in working directory<br>
+**git diff**: diff of what is changed but not staged<br>
+**git diff --staged**: diff of what is staged but not yet committed<br>
+**git commit -m “\[descriptive message\]”**: commit your staged content as a new commit snapshot<br>
+
+#### Branching and merging
```
Comment (JaydevSR) [inline README.md]:
```suggestion
### Branching and merging
```

File: README.md [hunk 5]
```diff
@@ -13,3 +13,37 @@
...
+**git branch**: list your branches. a * will appear next to the currently active branch<br>
+**git branch \[branch-name\]**: create a new branch at the current commit<br>
+**git checkout/git switch**: switch to another branch and check it out into your working directory<br>
+**git merge \[branch\]**: merge the specified branch’s history into the current one<br>
+**git log**: show the commit history for the currently active branch<br>
+**git log branchB..branchA**: show the commits on branchA that are not on branchB<br>
+**git diff branchB...branchA**: show the diff of what is in branchA that is not in branchB<br>
+
+#### Tracking path changes
```
Comment (JaydevSR) [inline README.md]:
```suggestion
### Tracking path changes
```

File: README.md [hunk 6]
```diff
@@ -13,3 +13,37 @@
...
+**git rm \[file\]**: delete the file from project and stage the removal for commit<br>
+**git mv \[existing-path\] \[new-path\]**: change an existing file path and stage the move<br>
+
+#### Sharing and updating
```
Comment (JaydevSR) [inline README.md]:
```suggestion
### Sharing and updating
```

File: README.md [hunk 7]
```diff
@@ -13,3 +13,37 @@
...
+**git remote add \[alias\] \[url\]**: add a git URL as an alias<br>
+**git fetch \[alias\]**: fetch down all the branches from that Git remote<br>
+**git merge \[alias\]/\[branch\]**: merge a remote branch into your current branch to bring it up to date<br>
+**git push \[alias\] \[branch\]**: Transmit local branch commits to the remote repository branch<br>
+**git pull**: fetch and merge any commits from the tracking remote <br>
+
+#### Rewrite history
```
Comment (JaydevSR) [inline README.md]:
```suggestion
### Rewrite history
```

File: README.md [hunk 8]
```diff
@@ -13,3 +13,37 @@
 ## Apendix
 
 ### Cheatsheet of git commands
+
+#### For setup and initialization
+**git init**: initialize an existing directory as a Git repository<br>
```
Comment (JaydevSR) [inline README.md]:
```suggestion
**`git init`**: initialize an existing directory as a Git repository<br>
```

Do this for all the commands

**Review by JaydevSR — CHANGES_REQUESTED** [review]
General formatting Changes

**Comment by JaydevSR** [comment]
@hritikb You can commit my suggestion as a batch from the files tab.

File: README.md [hunk 9]
```diff
@@ -13,3 +13,37 @@
...
+**git diff branchB...branchA**: show the diff of what is in branchA that is not in branchB<br>
+
+#### Tracking path changes
+**git rm \[file\]**: delete the file from project and stage the removal for commit<br>
+**git mv \[existing-path\] \[new-path\]**: change an existing file path and stage the move<br>
+
+#### Sharing and updating
+**git remote add \[alias\] \[url\]**: add a git URL as an alias<br>
+**git fetch \[alias\]**: fetch down all the branches from that Git remote<br>
+**git merge \[alias\]/\[branch\]**: merge a remote branch into your current branch to bring it up to date<br>
```
Comment (JaydevSR) [inline README.md]:
```suggestion
**git merge \[alias\]/ \[branch\]**: merge a remote branch into your current branch to bring it up to date<br>
```

**Review by hritikb — COMMENTED** [review]
Done

**Review by JaydevSR — APPROVED** [review]

**Pull request merged**